server/src/main/java/org/opensearch/cluster/service/ClusterManagerTaskThrottler.java

```diff
@@ -76,6 +60,21 @@ public ClusterManagerTaskThrottler(
         tasksThreshold = new ConcurrentHashMap<>(128); // setting initial capacity so each task will land in different segment
     }
 
+    /**
+     * To configure more task for throttling, override getClusterManagerThrottlingKey method with task name in task executor.
```

Comment:
[minor]: To configure a new task for throttling

Reply:
Ack.